fellow-submissions/Paul-Sobers/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Mansalva&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css" type="text/css" />
+
+  <title>Document</title>
+</head>
+<body>
+<header>
+  <h1>Welcome To Paul Gaming quiz</h1>
+  <p>In this quiz you will answer 5 questions about my favorite games.</p>
+  <img src="https://singularityhub.com/wp-content/uploads/2018/11/young-gamer-playing-video-game_shutterstock_624801452-1068x601.jpg">
+</header>
+<form action ="#" method="post">
+   <fieldset>
+    <fieldset class="game-name">
+      <ul>
+        <li>
+    <label for ="game">1: What is your fav COD?</label>
```

Comment:
The `for` attribute on your `<label>` should match the `id` for your `<input>`, not the `name` attribute.